chains/luksotestnet/metadata.yaml

```
@@ -0,0 +1,22 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://api.explorer.execution.testnet.lukso.network/api
+    family: blockscout
+    name: LUKSOExplorer
```

nit based on the name in the explorer home page
```suggestion
    name: LUKSO Testnet Explorer
```

---

Done